### ISSUE TITLE
Add lint job to CI GH Action

### ISSUE THREAD

**Issue by tschaffter** [data2health/nlp-sandbox-schemas#5]
Use `openapi-cli`

**Comment by tschaffter** [comment]
Install `openapi-cli` from `packages.json`.

```
$ npm install
$ npm -v openapi-cli
6.14.6
```

Lint one of our API locally:

```
$ npx @redocly/openapi-cli lint openapi/data-node/openapi.yaml
validating openapi/data-node/openapi.yaml...
openapi/data-node/openapi.yaml: validated in in 33ms

Woohoo! Your OpenAPI definition is valid. 🎉
```

or `openapi` if @redocly/openapi-cli has been installed globally:

```
$ openapi lint openapi/data-node/openapi.yaml
validating openapi/data-node/openapi.yaml...
openapi/data-node/openapi.yaml: validated in in 33ms

Woohoo! Your OpenAPI definition is valid. 🎉
```

**Comment by tschaffter** [comment]
Implemented in #6
